packages/integration-tests/src/test-project-setup/data_storage_auth_with_triggers.ts

```diff
@@ -133,7 +133,9 @@ class DataStorageAuthWithTriggerTestProject extends TestProjectBase {
       {
         sourceFile: sourceDataResourceFile,
         projectFile: dataResourceFile,
-        deployThresholdSec: 30,
+        // windows is consistently slower than other platforms
+        // https://github.com/microsoft/Windows-Dev-Performance/issues/17
+        deployThresholdSec: process.platform.startsWith('win') ? 40 : 30,
```

Comment:
I wonder if we should modify `TestProjectUpdate` type to reflect this.
I.e. have `deployThresholdSec = Record<string,number>` or perhaps add `deployThresholdWindowsSec` and have framework pick right value for assertion.

Reply:
I'm not strictly opposed, but having this ternary here vs an if statement somewhere else seems like 6 of one, half a dozen of another

Reply:
This is also currently the only test file that uses this value. Maybe a refactor like the one you're suggestion would make sense once we have multiple test classes that are using the deploy threshold

Reply:
That's because we have one test project we measure this.
If we get more we'll be choosing between "type reminds me that I should think about good value for windows" vs "when you copy this to next project remember to also copy that ternary for that property".

Reply:
> This is also currently the only test file that uses this value. Maybe a refactor like the one you're suggestion would make sense once we have multiple test classes that are using the deploy threshold

If we do this we should add code comment with `TODO` so that people who attempt to copy are at least warned and prompted for the refactor.

Reply:
updated with a refactor for your consideration :)

Reply:
thanks.